bug_report.php

```diff
@@ -302,6 +303,29 @@
 
 layout_page_header_begin();
 
+if ( ( ON === config_get_global( 'public_urls' ) ) && isset( $t_bug_data->token ) && ( !is_null( $t_bug_data->token ) ) ) {	
```

Comment:
Calculating the share URL should be an API rather than being embedded in script pages.  For example, bug_get_share_url( $p_bug_id ).  The API should handle the config checks, and return a valid URL or empty/null.